boards/arm/atsamr21_xpro/atsamr21_xpro.dts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2019 Benjamin Valentin
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <atmel/samd21.dtsi>
```

Comment:
We should include `atmel/samr21.dtsi`. The "soc: atmel: add SAMR21" commit should add the required .dtsi file.

Reply:
Ok.